package.json

```diff
@@ -1,10 +1,6 @@
 {
   "dependencies": {
-    "swagger-cli": "^2.2.1",
-    "swagger2openapi": "^5.2.5"
```

Comment:
We had `package.json` just to get `swagger-cli` and `swagger2openapi` (and run it), we can drop package.json completely. To prevent breaking of production builds https://github.com/RedHatInsights/notifications-backend/blob/master/.s2i/bin/assemble needs to be adjusted (or even removed completely)